shared/actions/settings.js

```diff
@@ -414,6 +414,17 @@ const unfurlSettingsSaved = (state, action) =>
       })
     )
 
+const loadHasRandomPW = state => {
```

Comment:
🐑 you could shorten this to `state => state.settings.passphrase.randomPW === null ? callRPCStuff() : null`